data/net/minecraft/core/Direction.mapping

```diff
@@ -6,17 +6,27 @@ CLASS net/minecraft/core/Direction
 	FIELD data3d I
 		COMMENT Ordering index for D-U-N-S-W-E
 	FIELD normal Lnet/minecraft/core/Vec3i;
-		COMMENT Normalized Vector that points in the direction of this Facing
+		COMMENT Normalized Vector that points in the direction of this Direction
 	FIELD oppositeIndex I
-		COMMENT Index of the opposite Facing in the VALUES array
+		COMMENT Index of the opposite Direction in the VALUES array
+	METHOD <init> (Ljava/lang/String;IIIILjava/lang/String;Lnet/minecraft/core/Direction$AxisDirection;Lnet/minecraft/core/Direction$Axis;Lnet/minecraft/core/Vec3i;)V
+		ARG 3 data3d
+		ARG 4 oppositeIndex
+		ARG 5 data2d
+		ARG 6 name
+		ARG 7 axisDirection
+		ARG 8 axis
+		ARG 9 normal
 	METHOD byName (Ljava/lang/String;)Lnet/minecraft/core/Direction;
-		COMMENT Get the facing specified by the given name
+		COMMENT @return the Direction specified by the given name or null if no such Direction exists
 		ARG 0 name
 	METHOD from2DDataValue (I)Lnet/minecraft/core/Direction;
-		COMMENT Gets the EnumFacing corresponding to the given horizontal index (0-3). Out of bounds values are wrapped around. The order is S-W-N-E.
+		COMMENT @return the Direction corresponding to the given horizontal index (0-3). Out of bounds values are wrapped around. The order is S-W-N-E.
+		COMMENT @see #get2DDataValue
 		ARG 0 horizontalIndex
 	METHOD from3DDataValue (I)Lnet/minecraft/core/Direction;
-		COMMENT Gets the EnumFacing corresponding to the given index (0-5). Out of bounds values are wrapped around. The order is D-U-N-S-W-E.
+		COMMENT @return the {@code Direction} corresponding to the given index (0-5). Out of bounds values are wrapped around. The order is D-U-N-S-W-E.
```

Comment:
Less of a request more of a comment. PR has inconsistent use of `{@code Foo}` vs. `Foo` when referring to specific classes. e.g. `Direction` vs. `{@code Direction}` vs. `direction` (lowercase being a reference to the word rather than the class).

Reply:
That is true and I am not sure which we should go for. I don't think the answer for which one we want is always the same either.

Reply:
In all honesty, me neither. I was thinking of this mostly from a standards PoV, but not sure if we *want* to enforce this, one way or the other. The only thing I'd say would be good is to make the distinction between i.e. `BlockPos` and `block position` - one refers to the class, the other to the concept.